bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/AbstractContainerImpl.java

```diff
@@ -67,9 +67,11 @@ public List<? extends ComponentExporter> getItems() {
     protected <T> List<T> getChildrenModels(@NotNull SlingHttpServletRequest request, @NotNull Class<T> modelClass) {
         List<T> models = new ArrayList<>();
         for (Resource child : slingModelFilter.filterChildResources(resource.getChildren())) {
-            T model = modelFactory.getModelFromWrappedRequest(request, child, modelClass);
-            if (model != null) {
-                models.add(model);
+            if (!child.getName().startsWith("fd:")) {
```

Comment:
Can we add this in SlingModeFilterImpl, so that customization is simple

Reply:
In the filter we need to add specific properties but we want all the properties that starts with `fd:`